src/app/modules/item/pages/item-details/item-details.component.html

```diff
@@ -18,15 +18,17 @@
         [routerLink]="'./'"
         routerLinkActive #contentTab="routerLinkActive"
         [routerLinkActiveOptions]="{exact: true}"
-        [class.active]="contentTab.isActive" i18n
-        [hidden]="contentTab.isActive && taskTabs.length > 0"
+        [class.active]="contentTab.isActive"
+        [hidden]="taskTabs.length > 0"
+        i18n
       >
         Content
       </a>
-      <ng-container *ngIf="contentTab.isActive && taskTabs.length > 0">
+      <ng-container *ngIf="taskTabs.length > 0">
```

Comment:
condition looks useless, if `taskTabs.length===0`, all should go well too

Reply:
Indeed, I removed it a bit after your comment :smile: